tasks/get_current_archive_list/get_current_archive_list.py

```diff
@@ -462,20 +463,30 @@ def get_s3_credentials_from_secrets_manager(secret_arn: str) -> tuple:
     return s3_access_key, s3_secret_key
 
 
+@dataclass(frozen=True)
+class MessageData:
+    """
```

Comment:
Add a line here on what it is with Args:
```
Data class that manages the message information needed to perform the task.

Args:
```

Reply:
Had to modify a bit to avoid styling warning around non-existent args.